src/components/open-api-page/hooks/use-on-click-outside.ts

```diff
@@ -25,7 +25,7 @@ function useOnClickOutside(ref, callbackFn) {
 		}
 		// Note: we expect "ref" to be a ref, which has no effect in a dependency array,
```

Comment:
If we added ref here then we should remove this comment as it's no longer relevant.